e2etest/base.mcf

```diff
@@ -0,0 +1,252 @@
+Database: google.com:datcom-store-dev.dc_v3_clustered
```

Comment:
Is this a copy of an existing mapping MCF? Can we reuse that?

Reply:
The existing mapping is a template of the BQ dataset. But since this does not test BQ query and probably not a good idea to test against real BQ, I just use the template mapping file. Removed the two files.